src/F2_autoreport.py

```diff
@@ -0,0 +1,119 @@
+    ],
+             style={
+                 'textAlign': 'left',
+                 'fontSize': '40px',
+                 'margin': '30px'
+             }),
+
+    html.Div(className='blockoftext1', children=[
+        html.Div(['The energy system with the ', html.Span(f'{projectName}', style={'fontStyle': 'italic'}),
+                  ' for the scenario ', html.Span(f'{scenarioName}', style={'fontStyle': 'italic'}), ' was simulated '
+                                                                                                     'with the '
+                                                                                                     'Multi-Vector '
+                                                                                                     'simulation tool '
+                                                                                                     'MVS 0.0x '
+                                                                                                     'developed from '
+                                                                                                     'the E-LAND '
+                                                                                                     'toolbox '
+                                                                                                     'developed in '
+                                                                                                     'the scope of '
+                                                                                                     'the Horizon '
+                                                                                                     '2020 '
+                                                                                                     'European '
+                                                                                                     'research '
+                                                                                                     'project. The '
+                                                                                                     'tool was '
+                                                                                                     'developed by '
+                                                                                                     'Reiner Lemoine '
+                                                                                                     'Institute and '
+                                                                                                     'utilizes the '
+                                                                                                     'OEMOF '
+                                                                                                     'framework.'])],
+
+             style={
```

Comment:
A bit too much text spreading no? :)

Reply:
Absolutely
This happened because of PyCharm. It kept insisting that the file needs to be reformted for proper paragraphing. I will change it back to the proper way and ignore the PyCharm warnings :)